code/modules/client/verbs/ooc.dm

```diff
@@ -385,3 +385,8 @@ GLOBAL_VAR_INIT(normal_ooc_colour, "#002eb8")
 		policytext += "No related rules found."
 
 	usr << browse(policytext.Join(""),"window=policy")
+
+/client/verb/fix_stat_panel()
+	set name = "Fix Stat Panel"
+
+	init_verbs()
```

Comment:
Maybe add a cooldown for this?

Reply:
I could set it to hidden instead, the browser automatically calls it when theres no verb_tabs or verbs

Reply:
Hmm, you could make the proc call having no `usr` ignore any cooldown as well, in case you want this to be an option for the users.
Up to you how you want to handle it.